doc/downsampling.md

```diff
@@ -50,7 +50,7 @@ Downsampling for prometheus counters will come soon.
 Downsampling is configured at the time of dataset creation. For example:
 
 ```
-./filo-cli -Dconfig.file=conf/timeseries-filodb-server.conf  --command create --dataset prometheus --dataColumns timestamp:ts,value:double --partitionColumns tags:map --shardKeyColumns __name__,_ns --downsamplers "tTime(0),dMin(1),dMax(1),dSum(1),dCount(1),dAvg(1)"
+./filo-cli -Dconfig.file=conf/timeseries-filodb-server.conf  --command create --dataset prometheus --dataColumns timestamp:ts,value:double --partitionColumns tags:map --shardKeyColumns __name__,_ns_,_ws_ --downsamplers "tTime(0),dMin(1),dMax(1),dSum(1),dCount(1),dAvg(1)"
```

Comment:
Oh, please remove all examples involving `create`, this is no longer a valid CLI command (configs are used only).